pages/en/lb4/sidebars/lb4_sidebar.yml

```diff
@@ -23,610 +23,683 @@ children:
   url: Inside-LoopBack-Application.html
   output: 'web, pdf'
 
-- title: 'Tutorials'
```

Comment:
I believe this file is not meant to be pushed?

Reply:
@dhmlau Yes I change it here for preview. The PR to change sidebar titles should be in lb-next.